elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/ElasticsearchConnectionSettings.scala

```diff
@@ -17,21 +22,40 @@ final class ElasticsearchConnectionSettings private (
 
   def hasCredentialsDefined: Boolean = username.isDefined && password.isDefined
 
+  def withHeaders(headers: List[HttpHeader]): ElasticsearchConnectionSettings =
```

Comment:
Please add a Java API for `withHeaders` that accepts a `java.util.Collection`

Reply:
The Java API should accept an `akka.http.javadsl.model.HttpHeader` too.

Reply:
I added this functionality.